Sources/Commands/SwiftTestTool.swift

```diff
@@ -288,6 +287,56 @@ public class SwiftTestTool: SwiftTool<TestToolOptions> {
                 executionStatus = .failure
             }
         }
+
+        if options.shouldEnableCodeCoverage {
+            let buildParameters = try self.buildParameters()
+            let fs = Basic.localFileSystem
+            mergeRawDataFiles(fs: fs, path: buildParameters.buildPath.appending(component: "codecov"))
+            if buildParameters.triple.isDarwin() {
```

Comment:
We should limit generating code coverage to Darwin. Instead, I would suggest that we compute paths to the coverage tools inside `UserToolchain` and make that computation based on `PATH` vs. `xcrun` depending on which platform we are on. I think we already have infrastructure for that in there.